content/en/metrics/explorer.md

```diff
@@ -51,12 +51,23 @@ The possible options are:
 
 You can optionally add functions to your query using the function button. Not all functions are available for all metric types. For more information, see the [querying][6] documentation. 
 
+### Split graph
+
+Use the **Split Graph** button to view the data split into individual graphs by tag values for tags such as region, service, or environment.
```

Comment:
To avoid repetition:
```suggestion
Use the **Split Graph** button to view the data split into individual graphs by values for tags such as region, service, or environment.
```